styles/variables-carbon.less

```diff
@@ -44,7 +44,7 @@
 @agate-panels-controls-top: (@agate-component-spacing - 6px);
 @agate-panels-panel-padding: 24px;
 @agate-panels-partial-height: 50%;
-@agate-panels-partial-width: auto;
+@agate-panels-partial-width: 890px;
```

Comment:
Why did you write a specific pixel number? Not 50%?

Reply:
I wrote a specific pixel number because I tried to maintain the same pattern we used in the other styles. e.g (styles/variables-gallium.less). Do you think it would be better to set it to 50%?

Reply:
No. I will merge this PR.